src/admin-guide/environments/gui.yml

```
@@ -0,0 +1,6 @@
+name: cylc-8-with-gui
+# Cylc 8 and the GUI installed from Conda
+channels:
+  - conda-forge
+dependencies:
+  - cylc
```

We are dropping the meta-package, the instructions on the [installation page](https://cylc.github.io/cylc-doc/latest/html/installation.html#quick-installation) are valid.

```suggestion
  - cylc-flow
  - cylc-uiserver
```